src/renderers/webxr/WebXRManager.js

```diff
@@ -172,6 +176,9 @@ class WebXRManager extends EventDispatcher {
 
 			//
 
+			renderer.setPixelRatio( currentPixelRatio );
+			renderer.setSize( currentSize.width, currentSize.height, false );
```

Comment:
Since you have mentioned it: These to lines should come after `scope.isPresenting = false;` otherwise `setSize()` produces a warning and does nothing.

Reply:
This is something I'd write a test for, but I'm not able to with QUnit currently.